quizroyale/db/errors.py

```diff
@@ -0,0 +1,4 @@
+class NoSuchUserException(Exception):
+    def __init__(self):
+        self.message = 'Invalid username or password!'
```

Comment:
Not needed

Reply:
Why? This msg could be shown to the user